src/encoded/static/scss/encoded/modules/_forms.scss

```diff
@@ -105,6 +113,18 @@ input[type="text"], input[type="password"], input[type="date"], input[type="date
 	}
 }
 
+.navbar-search-form {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
```

Comment:
These styles could potentially be added to `navbar-form`.

Reply:
I did not do it originally because navbar-form is part of twitter bootstrap and modifying its styles make updates even more complication. But I can if you like.

Reply:
It's good the way you wrote it!